packages/web-app-search/src/types.ts

```diff
@@ -1,18 +1,25 @@
-export interface SearchResult {
+export interface SearchResultValue {
   id: string
   data: unknown
 }
 
+export interface SearchResult {
+  meta?: {
```

Comment:
I thought that you and @fschade decided against a `meta` prop? 🤔 Is this PR up to date and ready for review?

Reply:
We had a call after that, and I show-case a proof of concept, @fschade was alright with that.

> Is this PR up to date and ready for review?
Yes